src/transformers/tokenization_utils_fast.py

```diff
@@ -109,34 +132,16 @@ def __init__(self, *args, **kwargs):
                 "have sentencepiece installed."
             )
 
+        tokenizer_converter, vocab_file = get_tokenizer_converter(kwargs.get("resolved_vocab_files", None), self.__class__.__name__)
```

Comment:
what if we are not converting and just want to load from `PreTrainedTokenizerFast`?